### ISSUE TITLE
NES: UNIF roms should not have .nes extension

### ISSUE THREAD

**Issue by tjanas** [SmokeMonsterPacks/EverDrive-Packs-Lists-Database#196]
[UNIF](https://wiki.nesdev.com/w/index.php/UNIF) is a different file format than [iNES](https://wiki.nesdev.com/w/index.php/INES).
The former should have a file extension of .unif, the latter should use .nes

**Comment by frederic-mahe** [comment]
That's easy to detect, first four bytes spell `UNIF`.
```sh
# quick and dirty
find . -type f | while read f ; do xxd -l 4 -c 4 "${f}" | grep -q "UNIF" && echo $f ; done
```
I find these, but my ROM set is a bit outdated.
```
./4 Hacks/Hacks D-J/Highway Star [hM00].nes
./4 Hacks/Hacks C/Contra/Contra_Fighter_(Unl)[!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/X-in-1 Untested/4-in-1 Digital Adventure (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/X-in-1 Untested/3-in-1 - Lion King 5, The (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/X-in-1 Untested/200-in-1 - Elfland (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/X-in-1 Untested/Shudu (5-in-1 + 15-in-1 + 138-in-1) (WX-105D, BS-5 PCB) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/X-in-1 Untested/2-in-1 - Sonic 5 + Earth Worm Jim 3 (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/3-in-1 - Harry Legend + Pokemon + Aladdin II [p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Super Mario Bros. Pocker Mali (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/2-in-1 - Aladdin 2 + Lion King, The [p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/City Fighter IV (Sound) (Unl) [U].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Pro Wrestling (FDS Conversion, LE05 Simplified) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Super Mario Bros Malee 2 (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/2-in-1 - Boogerman + Flintstones, The [p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Super Mario Bros. 2j (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Zanac (KAISER) (FDS Conversion) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Almana no Kiseki (Unl) (FDS Conversion) [U].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/2-in-1 - Mortal Kombat + Super Aladdin [p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/7-in-1 - Pokemon Yellow (HP10xx-HP20xx board)(KY7010)[p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Gyruss (FDS Conversion,Kaiser) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/4-in-1 - Chessmaster [p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Bubble Bobble (FDS Conversion) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Super Mario Bros. 13 (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Bubble Bobble (FDS Conversion, CHR-ROM version) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/2-in-1 - Sonic 5 + Batman 3 (YH226)[p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Monty no Doki Doki Daisassou (FDS Conversion) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/4-in-1 - Digital Monster [p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/6-in-1 - Spider Man 2 (HP10xx-HP20xx board)[p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/7-in-1 - Spider Man 2 (HP10xx-HP20xx board)(KY7005)[p1][U][b1] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Fuuun Shaolin Kyo (FDS Conversion) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/4-in-1 - Harry Legend (YH3139)[p1][U][!] (Unl).nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/Green Beret (FDS Conversion, LH09) (Unl) [U][!].nes
./2 Unlicensed - A-Z/Unsupported or Glitches/Non-Booting/6-in-1 - Lost World, The (HP10xx-HP20xx board)[p1][U][!] (Unl).nes
./Unsupported/N8 - Too Large/9-in-1 King001 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/100-in-1 Arcade Action II (AT-103) (Unl) [U][!].nes
./Unsupported/N8 - Too Large/DreamGEAR 75-in-1 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/23 Plus 222-in-1 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/17-in-1 (60311C) (Unl) [U].nes
./Unsupported/N8 - Too Large/Power Joy Supermax 30-in-1 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/Power Joy Supermax 60-in-1 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/Zelda III (ES-1096)(Ch)[U][!].nes
./Unsupported/N8 - Too Large/100-in-1 (D-CAT8 Console, v5.01.11-frd, BL 20041217) (Unl) [U][!].nes
./Unsupported/N8 - Too Large/Mei Shao Nv Meng Gong Chang (Princess Maker) (Sachen) (Unl) [U][!].nes
./Unsupported/N8 - Too Large/Legend of Zelda - Triforce of the Gods (C) T+Eng v1 Pacnsacdave.nes
./Unsupported/N8 - Too Large/45-in-1 (JY-120A) (Unl) [U][!].unf
./Unsupported/N8 - Too Large/Power Joy Navigator 50-in-1 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/He Jin Zhuang Bei 150-in-1 Real Game (Unl) [U][!].nes
./Unsupported/N8 - Too Large/Sports Game 69-in-1 (Unl) [U][!].nes
./Unsupported/N8 - Too Large/2-in-1 - Street Dance + Hit Mouse (Unl) [U][!].nes
./Unsupported/N8 - Too Large/6-in-1 Rockman (Unl) [U][!].nes
```

**Comment by frederic-mahe** [comment]
fixed by commit fab6290627c4f2b45fc4d0a0928485d06622bf39
